test/distributed/org/apache/cassandra/fuzz/topology/AccordTopologyMixupTest.java

```diff
@@ -240,13 +225,52 @@ public String createSchema()
 
     private static class AccordState extends State<Spec>
     {
+        private final Map<Integer, String> instanceEpochReadyState = new TreeMap<>();
+        private final Map<Integer, String> instanceEpochSyncState = new TreeMap<>();
```

Comment:
collect the results of the 2 new table *before* each command, so we know the state before the command ran.